mitodl/mit-open
# testimonial fixtures

### What are the relevant tickets?

Closes https://github.com/mitodl/hq/issues/4687

### Description (What does it do?)
This PR adds a data migration to populate testimonials from the current production configuration

### How can this be tested?
Checkout this branch, and run migrations. Testimonials should be visible on the homepage and copy should match what is on production.

### Additional Context
The introduction of this data migration is part of a discussion around how we should be going about doing one-off static data migrations. alternate implementation and notes https://github.com/mitodl/mit-open/pull/1163
